### PR TITLE
Misc styling changes

### PR DESCRIPTION
**Pull request by dannymcgee**
- Centered the main container
- Added some subtle global transitions
- Added hover states and cursor: pointer for buttons
- Added some extra horizontal padding for buttons
- Darkened the default link color on Light theme for WCAG AA compliance

### CODE REVIEW

File: water.css [hunk 1]
```diff
@@ -3,11 +3,22 @@ body {
...
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   text-rendering: optimizeLegibility;
-  margin: 20px;
+  margin: 20px auto;
   max-width: 800px;
   line-height: 1.4;
 }
 
+button, input[type="submit"] {
+  cursor: pointer;
```
Comment (keegs112) [inline water.css]: This selector is declared twice. I'd move `cursor: pointer;` down to the selector on line 71.

**Comment by kognise** [comment]
I need to set up some way to preview changes. Any ideas? Thanks for the PR btw.

**Comment by kognise** [comment]
I moved the demo out of the docs folder for easier development - I checked, github pages will prefer index.html over README.md.

**Comment by silvestreh** [comment]
> I need to set up some way to preview changes. Any ideas? Thanks for the PR btw.

You could enable brandh deploys to netlify and set `docs/index.html` as the entry point.

**Comment by kognise** [comment]
@silvestreh Now I've moved the demo out of the docs folder so local development can be done with a simple static server- I might switch to Netlify at some point! Thanks.

**Comment by kognise** [comment]
Thank you so much! Merging.

**Comment by kognise** [comment]
:shipit:

**Pull request merged**